File: src/main/java/rbq/wtf/lycoris/client/wrapper/wrappers/wrapper/utils/event/HoverEvent.java
package rbq.wtf.lycoris.client.wrapper.wrappers.wrapper.utils.event;

import rbq.wtf.lycoris.client.wrapper.MapEnum;
import rbq.wtf.lycoris.client.wrapper.wrappers.annotation.WrapperClass;
import rbq.wtf.lycoris.client.wrapper.wrappers.wrapper.IWrapper;

@WrapperClass(mcpName = "net.minecraft.event.HoverEvent",targetMap = MapEnum.VANILLA189)
public class HoverEvent extends IWrapper {
    public HoverEvent(Object obj) {
        super(obj);
    }
}

File: src/main/java/rbq/wtf/lycoris/client/wrapper/wrappers/util/event/HoverEvent.kt
package rbq.wtf.lycoris.client.wrapper.wrappers.util.event

import rbq.wtf.lycoris.client.wrapper.IWrapper
import rbq.wtf.lycoris.client.wrapper.MapEnum
import rbq.wtf.lycoris.client.wrapper.annotation.WrapperClass

@WrapperClass(mcpName = "net.minecraft.event.HoverEvent", targetMap = MapEnum.VANILLA189)
class HoverEvent(obj: Any) : IWrapper(obj)